src/local.rs

```diff
@@ -41,23 +34,23 @@ impl LocalAsset {
         match fs::write(&dist_path, &self.contents) {
             Ok(_) => Ok(dist_path),
             Err(details) => Err(AxoassetError::LocalAssetWriteFailed {
-                asset: self.to_string(),
+                origin_path: self.origin_path.to_string(),
                 dist_path: dist_path.display().to_string(),
                 details: details.to_string(),
             }),
         }
     }
 
-    pub fn copy(origin_path: &str, dist_dir: &str, label: &str) -> Result<PathBuf> {
-        LocalAsset::load(origin_path, label)?.write(dist_dir)
+    pub fn copy(origin_path: &str, dist_dir: &str) -> Result<PathBuf> {
```

Comment:
ok yeah some kinda type alias to make all the strings more distinctive might indeed a good idea

Reply:
filed #10